examples/dgmulti_3d/elixir_navierstokes_convergence_curved.jl

```diff
@@ -35,211 +37,225 @@ mesh = DGMultiMesh(dg, cells_per_dimension, mapping; periodicity=(true, false, t
-                              - mu_ * (tau11_x + tau12_y + tau13_z) )
-  # y-momentum equation
-  du3 = ( rho_t * v2 + rho * v2_t + p_y + rho_x * v1    * v2
-                                        + rho   * v1_x  * v2
-                                        + rho   * v1    * v2_x
-                                        +         rho_y * v2^2
-                                        + 2.0   * rho   * v2 * v2_y
-                                        + rho_z * v2    * v3
-                                        + rho   * v2_z  * v3
-                                        + rho   * v2    * v3_z
-                              - mu_ * (tau12_x + tau22_y + tau23_z) )
-  # z-momentum equation
-  du4 = ( rho_t * v3 + rho * v3_t + p_z + rho_x * v1    * v3
-                                        + rho   * v1_x  * v3
-                                        + rho   * v1    * v3_x
-                                        + rho_y * v2    * v3
-                                        + rho   * v2_y  * v3
-                                        + rho   * v2    * v3_y
-                                        +         rho_z * v3^2
-                                        + 2.0   * rho   * v3 * v3_z
-                              - mu_ * (tau13_x + tau23_y + tau33_z) )
-  # Total energy equation
-  du5 = ( E_t + v1_x * (E + p) + v1 * (E_x + p_x)
-              + v2_y * (E + p) + v2 * (E_y + p_y)
-              + v3_z * (E + p) + v3 * (E_z + p_z)
-    # stress tensor and temperature gradient from x-direction
-      - mu_ * ( q_xx + v1_x * tau11   + v2_x * tau12   + v3_x * tau13
-                     + v1   * tau11_x + v2   * tau12_x + v3   * tau13_x)
-    # stress tensor and temperature gradient terms from y-direction
-      - mu_ * ( q_yy + v1_y * tau12   + v2_y * tau22   + v3_y * tau23
-                     + v1   * tau12_y + v2   * tau22_y + v3   * tau23_y)
-    # stress tensor and temperature gradient terms from z-direction
-      - mu_ * ( q_zz + v1_z * tau13   + v2_z * tau23   + v3_z * tau33
-                     + v1   * tau13_z + v2   * tau23_z + v3   * tau33_z) )
-
-  return SVector(du1, du2, du3, du4, du5)
+    # TODO: parabolic
+    # we currently need to hardcode these parameters until we fix the "combined equation" issue
+    # see also https://github.com/trixi-framework/Trixi.jl/pull/1160
+    inv_gamma_minus_one = inv(equations.gamma - 1)
+    Pr = prandtl_number()
+    mu_ = mu()
+
+    # Constants. OBS! Must match those in `initial_condition_navier_stokes_convergence_test`
+    c = 2.0
+    A1 = 0.5
+    A2 = 1.0
+    A3 = 0.5
+
+    # Convenience values for trig. functions
+    pi_x = pi * x[1]
+    pi_y = pi * x[2]
+    pi_z = pi * x[3]
+    pi_t = pi * t
+
+    # Define auxiliary functions for the strange function of the y variable
+    # to make expressions easier to read
+    g = log(x[2] + 2.0) * (1.0 - exp(-A3 * (x[2] - 1.0)))
+    g_y = (A3 * log(x[2] + 2.0) * exp(-A3 * (x[2] - 1.0))
```

Comment:
Line breaking plus and minus signs.

Reply:
Can you expand on this? I don't see the line break or `+`/`-` signs you're referring to.

Reply:
From line 86 on.